### DISCUSSION TITLE
No time in Tooltip , only date

### DISCUSSION THREAD

**Discussion by masterwishx** [grafana/grafana#50144] [Dashboards, Panels & Visualizations]
I have only date in tooltip for all dashboars , how to add time also ?

![image](https://user-images.githubusercontent.com/28630321/171821857-16628398-3cca-454e-b2ea-a03bb13e081c.png)

**Comment by masterwishx** [comment]
Fixed by changed full_date in config

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
